packages/rgbpp/src/rgbpp/types.ts

```diff
@@ -0,0 +1,25 @@
+import { DataSource } from '@rgbpp-sdk/btc';
+import { BtcTransferVirtualTxResult, Collector, Hex } from '@rgbpp-sdk/ckb';
+
+export interface RgbppTransferTxParams {
+  // The collector that collects CKB live cells and transactions
+  collector: Collector;
+  // The transferred RGB++ xUDT type script args
+  xudtTypeArgs: Hex;
+  // The rgbpp assets cell lock script args array whose data structure is: out_index | bitcoin_tx_id
+  rgbppLockArgsList: Hex[];
+  // The XUDT amount to be transferred, if the noMergeOutputCells is true, the transferAmount will be ignored
+  transferAmount: bigint;
+  // The sender BTC address
+  fromBtcAddress: string;
```

Comment:
For P2TR inputs, the `fromPubkey` prop is required in the `sendRgbppUtxos()` API. BTW, do you think the `feeRate` prop should be exposed, too? ref: https://github.com/ckb-cell/rgbpp-sdk/pull/223/files#diff-45996e7b4d331bb196c3c3af127807387136cfdc33ff6de8cc604b40221b2b46R41

Reply:
I just want to keep the parameters simple,  and we can discuss about which parameters are needed.

Adding `fromPubkey` and `feeRate` to parameters is OK for me.

Reply:
Updated in https://github.com/ckb-cell/rgbpp-sdk/pull/216/commits/7bd4e7f40236d7d804d4a4085e85a5b6ed38c0ad